README.md

```diff
@@ -88,10 +82,13 @@ It supports the popular IDEs for managing GraphQL requests and exploring GraphQL
 
 ## Upgrade Guides
 
-* 2.4.0 to 3.0 - under development
+* [3.x to 4.x](https://graphql-dotnet.github.io/docs/guides/migration4)
```

Comment:
Let it be here even page does not exist yet.

Reply:
```suggestion
* [3.x to 4.x](https://github.com/graphql-dotnet/graphql-dotnet/blob/develop/docs2/site/docs/guides/migration4.md)
```

Reply:
Let's make the link actually work at least.

Reply:
OK I didn't think about it.